### PR TITLE
chore(bpmn): add immediate timer re-evaluation section

### PR DESCRIPTION
**Pull request by yanavasileva**
+ adjust rest link

https://github.com/camunda/camunda-bpm-platform/issues/4008

### CODE REVIEW

File: content/reference/bpmn20/events/img/change-timer-due-date.png
Comment (mboskamp) [inline content/reference/bpmn20/events/img/change-timer-due-date.png:1]: 💭 I think this screenshot would be easier to read if we cut it to only the modal box. What do you think?

File: content/reference/bpmn20/events/timer-events.md [hunk 1]
```diff
@@ -124,6 +124,20 @@ At 4 p.m., after the timer has fired two times, we adjust the bean to return `my
 
 The feature is disabled by default. To enable it, set the `reevaluateTimeCycleWhenDue` property to `true` in the process engine configuration.
 
+#### Immediate timer re-evaluation
+
+To enforce an immediate re-evaluation of a time cycle, follow the steps:
+
+1. Change the expression of the time cycle. (E.g.: Adjust the Spring bean that resolves the cycle.)
```
Comment (mboskamp) [inline content/reference/bpmn20/events/timer-events.md:131]: 🙃 
```suggestion
1. Change the expression of the time cycle (e.g., Adjust the Spring bean that resolves the cycle).
```

**Review by mboskamp — APPROVED** [review]
❓ In the issue, adding an "info box" is specifically mentioned. Is this a hard requirement?

**Comment by yanavasileva** [comment]
> ❓ In the issue, adding an "info box" is specifically mentioned. Is this a hard requirement?

I reached to Tobi and decide to have an info box without a screenshot (https://github.com/camunda/camunda-bpm-platform/issues/4008#issuecomment-1903660952).

**Review by mboskamp — APPROVED** [review]

**Pull request merged**